### ISSUE TITLE
reproduce D0 performance on COCO2017

### ISSUE THREAD

**Issue by KaqiKaqi** [zylo117/Yet-Another-EfficientDet-Pytorch#43]
Hi, thanks for your excellent work!
I really want to reproduce your result on coco2017 using D0. Could you please tell me the training setting parameters, such as number of epochs, batch_size(is it batch size on each gpu or batch size in total?), and initial learning rate?

Besides, what does it mean by 'This takes time and requires change of hyperparameters every few hours.' Do we need to tune hyperpamaters by ourselves?

Thanks a lot again : )

**Comment by claire-s11** [comment]
Hi, thanks for your work! I'm also trying to reproduce the result on COCO 2017 with D0. I plan to train on COCO 2017 from scratch. Is it possible to provide your training log? 
@KaqiKaqi , have you started training? Just to let you know that the author did some changes on the loss function, you might want to pull the latest code before training.

**Comment by Li505358678** [comment]
@KaqiKaqi  @claire-s11 Do you have good result on coco2017?If you do ,how do you do?

**Comment by yangbisheng2009** [comment]
Do you have a good result on coco2017?
I use this with my dataset. But I can not get a good result.

**Comment by KaqiKaqi** [comment]
@yangbisheng2009 @Li505358678 @claire-s11 I have stopped trying to train on coco, instead I'm studying official implementation now : )

**Comment by yangbisheng2009** [comment]
yeah，this implementation is not robust
